### PR TITLE
Add header

### PR DESCRIPTION
**Pull request by akos29**
### Create Page Header

In this request: 

- Add header section 
- Style the header section for #17

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,9 +3,27 @@
...
+                <li class="links"><a href="#">TV Shows</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+
+    </main>
+    <footer>
+
+    </footer>
```
Comment (krisbobo) [inline src/index.html:27]: Good job using semantic HTML tags.

File: webpack.config.js [hunk 2]
```diff
@@ -22,36 +22,46 @@ module.exports = {
...
+  //     react: './vendor/react-master',
+  //   },
+  //   extensions: ['.js', '.jsx'],
+  //   modules: [
+  //     'node_modules',
+  //     'bower_components',
+  //     'shared',
+  //     '/shared/vendor/modules',
+  //   ],
+  // },
```
Comment (krisbobo) [inline webpack.config.js]: I suggest you remove all commented code in your files. Removing them will help improve code readability.

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,55 @@
...
+
+.nav li {
+  display: var(--inline);
+  padding: 2rem 3rem;
+  font-size: 1.5rem;
+  font-weight: 600;
+}
+
+.nav li:hover {
   background-color: white;
```
Comment (krisbobo) [inline src/style.css]: I suggest you reuse the variable you declared above. This ensures you adhere to CSS best practices as repetition is avoided(DRY code)

**Review by krisbobo — CHANGES_REQUESTED** [review]
## Changes Required ♺
**Hi @akos29** 
Good job implementing some of the project requirements:
- Linters passing✔️
- HTML best practices ✔️

   *Consider applying the following suggestions below:*

File: src/style.css (hunk 3 above)
Comment (akos29) [reply src/style.css]: Updated accordingly. 🙏

File: webpack.config.js (hunk 2 above)
Comment (akos29) [reply webpack.config.js]: Some are removed some are used. 😉

File: src/index.html (hunk 1 above)
Comment (akos29) [reply src/index.html:27]: Thanks!

**Review by krisbobo — APPROVED** [review]
## Status: Approved✅✅
*Hi @akos29* 
The project is complete. It is time to merge it.

**Pull request merged**